activerecord/lib/active_record/transactions.rb

```diff
@@ -11,6 +11,7 @@ module Transactions
                        :before_commit_without_transaction_enrollment,
                        :commit_without_transaction_enrollment,
                        :rollback_without_transaction_enrollment,
+                       terminator: deprecated_false_terminator(ActiveRecord::Transactions),
```

Comment:
Ehehe, I thought about that and I wasn't sure which one to pick.

Since it confuses you, I'd say let's go with `ActiveRecord` instead!